### ISSUE TITLE
Fully-grown, wild pineapples do not always drop pineapples when harvested/broken.

### ISSUE THREAD

**Issue by kurazarrh** [anegostudios/VintageStory-Issues#2198]
**Game Version:** 1.17.3 and 1.17.4
**Platform:** Windows    
**Modded:** No
**SP/MP:** Singleplayer

### Description
Fully-grown, wild pineapples do not always drop pineapples.

### How to reproduce
1. Find some wild-growing, mature pineapples that are at 16/16 growth stage.
2. Break the pineapples. Recommended to break 10-20 of them.
3. You get fewer pineapples than mature plants that you harvested. In my experience, I've only gotten about 50% of the pineapples from wild plants that I should have.

### Expected behavior
Harvesting a fully-grown, wild pineapple should always provide 1 pineapple, in addition to any seeds.

**Comment by kurazarrh** [comment]
As of 1.17.4, this or a similar issue appears to affect most or all wild crops (no word yet on farmed crops, but will check back in whenever my farm grows). Breaking amaranth or soybeans, I have gotten 1 (75%) or 2 (25%) drops from each crop. Same is true for turnips and spelt (the only other crops I have found so far since this issue started appearing). Cassava seems to be exempt--harvesting wild cassava plants is giving me ~4 crop drops.

Reporting all this as a bug, since I did not notice this stark reduction mentioned in the patch notes. If the contents of this comment constitute a separate bug, let me know and I can split it if necessary.
